be/src/controlers/rest.router.ts

```diff
@@ -1,22 +1,53 @@
-// @ts-ignore
 import {Request, Response, Router} from 'express';
+import {vocabularyService} from "../services/vocabulary.service";
+import {Vocabulary} from "../models/dbo.models";
+import {Vocabulary as VocabularyDTO} from "../generated/models/Vocabulary";
```

Comment:
Why not naming the own generated DTO models in the `dto.models.ts` file
https://github.com/TailoredMediaProject/tm-knowledge-store/pull/9/files#diff-a0dc0cd6a924352272536c9dbef15c3be3cc6b7c3030a839446b524a2b568a7e
with the suffix DTO and leaving the created ones as they are? The renaming for this and for all other files in the future is overly complicated

Reply:
`DTO` stands for **D**ata**T**ransfer**O**bject in contrast to `DBO` (**D**ata**b**ase**O**bject)

So either: 
- we manage to generate the Api-Models with the `DTO`-suffix, or 
- we add a `DBO`-suffix to the manual definitons.
- keep it as-is

I'd prefer (1) or (3), since the DTOs should be only used on the API-Layer.

Reply:
(3) :)